application/common/models/User.php

```diff
@@ -343,6 +343,55 @@ private function savePassword()
         return true;
     }
 
+    public static function getExpiringUsers($criteria): array
```

Comment:
This sounds like a question of clarity around what/how `$criteria` is to be used.  Docs should help with that, see https://github.com/silinternational/idp-id-broker/commit/c79866937c503de1b5d60cb4a75d358f146cec5b